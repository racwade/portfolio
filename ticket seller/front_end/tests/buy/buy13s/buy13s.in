login
test_sellrich
logout