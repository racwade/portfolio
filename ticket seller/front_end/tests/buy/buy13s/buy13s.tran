00 test_sellrich       SS 000999999