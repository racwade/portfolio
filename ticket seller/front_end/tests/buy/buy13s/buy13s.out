welcome to the ticket seller system
enter username:
logged in as test_sellrich
bye